src/internal/connector/exchange/api/contacts.go

```diff
@@ -55,6 +55,41 @@ func (c Contacts) CreateContactFolder(
 	return mdl, nil
 }
 
+// GetContactFolderByDisplayName fetches contact folder which has the displayName of
+// folderName. This is only done at root level.
+func (c Contacts) GetContactFolderByDisplayName(
+	ctx context.Context,
+	user, folderName string,
+) (models.ContactFolderable, error) {
+	filter := fmt.Sprintf("displayName eq '%s'", folderName)
+	options := &users.ItemContactFoldersRequestBuilderGetRequestConfiguration{
+		QueryParameters: &users.ItemContactFoldersRequestBuilderGetQueryParameters{
+			Filter: &filter,
+		},
+	}
+
+	resp, err := c.Stable.Client().UsersById(user).ContactFolders().Get(ctx, options)
+	if err != nil {
+		return nil, graph.Wrap(ctx, err, "getting contact folder").
+			With("folder name", folderName)
```

Comment:
no spaces in structured keys, please.  Use snake_case by default.  
```suggestion
		return nil, graph.Wrap(ctx, err, "searching contact folder by name").
			With("folder_name", folderName)
```